gaia-fast-list.js

```diff
@@ -894,6 +913,7 @@ Internal.prototype = {
   cacheImage(image, index) {
     if (!this.el.imageCacheLength) return;
     if (!this.el.imageCacheSize) return;
+    if (this.images.hash[index]) return;
```

Comment:
can you rebase this one on master to trigger a new build and see if the intermittent goes away?